src/config/tours.js

```diff
@@ -0,0 +1,75 @@
+import {
+  PHASE_DRAWER_CLASS,
+  SPACE_PREVIEW_ICON_CLASS,
+  SPACE_SCREEN_CLASS,
+  VISIT_SPACE_BUTTON_CLASS,
+  VISIT_SPACE_INPUT_CLASS,
+  SETTINGS_ACTIONS_CLASS,
+  LANGUAGE_SELECT_ID,
+  STUDENT_MODE_SWITCH_ID,
+} from './selectors';
+
+const tours = {
+  VISIT_SPACE_TOUR: 'visitSpace',
+  SETTINGS_TOUR: 'settings',
+};
+
+const VISIT_SPACE_TOUR_STEPS = [
+  {
+    target: `.${VISIT_SPACE_BUTTON_CLASS}`,
+    title: 'Welcome to Graasp Desktop!',
+    content: "Let's get started by visiting a space and saving it locally.",
+    disableBeacon: true,
+  },
+  {
+    target: `.${VISIT_SPACE_INPUT_CLASS}`,
+    title: 'Visiting a Space',
+    content:
+      "You can visit a space using its URL or ID. For example, let's have a look at space https://graasp.eu/s/owozgj or simply owozgj.",
```

Comment:
You can use the factored out value of the sample space here too.

Reply:
Sorry I did not understand your proposal. What do you mean?

Reply:
```
content: `You can... have a look at space https://...${TOUR_SPACE}... or ${TOUR_SPACE}`
```